docs/application/dotnet/guides/nui/grid-layout.md

```diff
@@ -0,0 +1,37 @@
+# Grid Layout
+
+## Overview
+
+`GridLayout` is a grid box for the two-dimensional layout. It constraints the x and y position, width, and height of the child actors.
+
+Positioning children in a grid form, the cells are of uniform size based on the first child added to the parent View.
+
+The number of **columns** can be specified and the rows will automatically increased to hold the children. Once the available space is used up, further rows will not be added.
+
+| Property               | Type            | Description |
+| -----------------------| --------------- | ------------ |
+| `Columns`              | int             | Gets/Sets the number of columns in the Grid |
+
+
+## Column
+
+<table style="width:100%">
+<tr>
+<td style="width:100%" align="center">
+<img src="./media/columnLayout.png">
+</td>
+</tr>
+</table>
```

Comment:
Global comment:

Kindly use MD format to display images. For more infomation, see: https://github.com/Samsung/tizen-docs/blob/master/styleguide/style.md#markdown-style-guide

Reply:
Done.